compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEUnrollAndDistributeWorkgroup.cpp

```diff
@@ -166,7 +166,13 @@ class AMDAIEUnrollWorkgroupLoops : public OpRewritePattern<scf::ForOp> {
     int64_t lbInt = getConstantIntValue(forOp.getLowerBound()).value();
     int64_t ubInt = getConstantIntValue(forOp.getUpperBound()).value();
     int64_t stepInt = getConstantIntValue(forOp.getStep()).value();
-    if (lbInt != 0 || stepInt != 1) return failure();
+    // TODO(avarma): Either :-
+    //               1. Enforce loop normalisation before this pass.
```

Comment:
I was looking at the same :s I almost have it so will push shortly and you could have a look

Reply:
Took a bit more time. Initially, I implemented it for `scf.for`, but actually it's needed on `scf.forall` before the `amdaie.core` operations are inserted. Here is the PR though: https://github.com/nod-ai/iree-amd-aie/pull/375

Reply:
Aah okay. No worries. I'll close this PR. Taking a look at your PR now.